### PR TITLE
WEBEU2 - User Interface - Wasiu Idowu 🇳🇬

### PR DESCRIPTION
**Pull request by Hoxtygen**
This commit fixes the alleged bug in chrome which was found out when writing same code in another branch. This now renders the same way in firefox.
What was thought to be a bug isn't actually a bug, the weird rendering got fixed when VSCode  live server was restarted. As to why live server caused that behaviour is still unknown for now.

### CODE REVIEW

File: great-idea-website/index.html [hunk 1]
```diff
@@ -1,86 +1,84 @@
-<!doctype html>
-
+<!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="utf-8">
-
-  <title>Great Idea!</title>
-
+  <meta charset="UTF-8">
```
Comment (maximesalomon) [inline great-idea-website/index.html:4]: Great!

File: great-idea-website/index.html [hunk 2]
```diff
@@ -1,86 +1,84 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Lambda Test</title>
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
```
Comment (maximesalomon) [inline great-idea-website/index.html:8]: Google fonts 😍

File: great-idea-website/index.html [hunk 3]
```diff
@@ -1,86 +1,84 @@
...
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+        <div id="logo">
+            <div class="image-container">
+                <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+            </div>
+        </div>
     </header>
```
Comment (maximesalomon) [inline great-idea-website/index.html:31]: Great use of semantic tag - header

File: great-idea-website/index.html [hunk 4]
```diff
@@ -1,86 +1,84 @@
...
     <main>
```
Comment (maximesalomon) [inline great-idea-website/index.html:32]: Same here - main for the win

File: great-idea-website/index.html [hunk 5]
```diff
@@ -1,86 +1,84 @@
...
+                <br>
+                Demand
+            </h1>
+           <a href="#" class="ctn"> Get Started</a>
           </div>
-          <div id="about">
-            <h3>About</h3>
-            <p>  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          <div class="right-container">
+              <img src="img/header-img.png" alt="Image of a code snippet." >
```
Comment (maximesalomon) [inline great-idea-website/index.html:45]: It's important to use alt - good

File: great-idea-website/index.html [hunk 6]
```diff
@@ -1,86 +1,84 @@
...
-          </div>
-        </div>
-        <div class="bottom-container box">
-          <div id="services">
-            <h3>Services</h3>
-            <p>  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </section>
+        <hr>
+        <section class="nav-container">
+          <div class="top-container box">
```
Comment (maximesalomon) [inline great-idea-website/index.html:50]: Good use of section + naming

File: great-idea-website/css/style.css [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
```
Comment (maximesalomon) [inline great-idea-website/css/style.css:65]: @Hoxtygen Do you understand why we use a CSS reset here?

File: great-idea-website/css/style.css [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+/* Your code starts here! */
+
```
Comment (maximesalomon) [inline great-idea-website/css/style.css:67]: I would recommend positioning the bigger box first. Here you are missing body and therefore the content doesn't behave the same for navigation and main for example

File: great-idea-website/css/style.css [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+a   {
+    color: #222;
+    text-decoration: none;
+}
```
Comment (maximesalomon) [inline great-idea-website/css/style.css:71]: You might want to add a hover style to your links - looks way better

File: great-idea-website/css/style.css [hunk 10]
```diff
@@ -0,0 +1,204 @@
...
+    color: grey;
+    font-weight: 500;
+
+}
+
+main    {
+    width: 85%;
+    margin: 0 auto;
+    padding: 20px;
+}
```
Comment (maximesalomon) [inline great-idea-website/css/style.css:113]: You might want to add a max-width in px to make sure it looks good as well on bigger screen

File: great-idea-website/css/style.css [hunk 11]
```diff
@@ -0,0 +1,204 @@
...
+
+.image-container img	{
+	display: block;
+	margin: auto;
+	width: 100%;
+}
+
+.statement-text 	{
+	font-size: 100px;
+}
```
Comment (maximesalomon) [inline great-idea-website/css/style.css:138]: Take a look at em for font-sizing

File: great-idea-website/css/style.css [hunk 12]
```diff
@@ -0,0 +1,204 @@
...
+
+section 	{
+	padding-top: 40px;
+	padding-bottom: 40px;
+}
```
Comment (maximesalomon) [inline great-idea-website/css/style.css:143]: Try to avoid putting styling on not very specific tags

File: great-idea-website/css/style.css [hunk 13]
```diff
@@ -0,0 +1,204 @@
...
+
+#about 	{
+	display: inline-block;
+	width: 49%;
+}
+
+#services 	{
+	display: inline-block;
+	width: 33%;
+}
```
Comment (maximesalomon) [inline great-idea-website/css/style.css:178]: You can add some padding here to make the text looks good

**Review by maximesalomon — APPROVED** [review]
Good job Wasiu! There a few improvements you can make but it looks good :)

File: great-idea-website/css/style.css (hunk 10 above)
Comment (Hoxtygen) [reply great-idea-website/css/style.css:113]: okay, will do this in the next project.

File: great-idea-website/css/style.css (hunk 7 above)
Comment (Hoxtygen) [reply great-idea-website/css/style.css:65]: Yes.

**Pull request merged**